File: ManagerApplication/ManagerApp/src/app/model/renovation.ts
export class Renovation {
    id:number;
    firstRoomId: number;
    secondRoomId: number;
    Date: Date;
    Duration: number;
    Type: TypeOfRenovation;

    constructor(id: number, firstRoomId: number, secondRoomId: number, RenovationType: TypeOfRenovation, Date: Date, 
        Duration: number){
        this.id = id;
        this.firstRoomId = firstRoomId;
        this.secondRoomId = secondRoomId;
        this.Type = RenovationType;
        this.Date = Date;
        this.Duration = Duration;
    }
}

export enum TypeOfRenovation{
    Divide,
    Merge
}

File: HealthCareClinic/Hospital/Rooms and equipment/Model/Renovation.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace Hospital.Rooms_and_equipment.Model
{
    public class Renovation
    {

        public enum RenovationType
        {
            Divide,
            Merge
        }
        public int Id
        {
            get; set;
        }

        public int FirstRoomId
        {
            get; set;
        }

        public int SecondRoomId
        {
            get; set;
        }

        public DateTime Date
        {
            get; set;
        }

        public int Duration
        {
            get; set;
        }
        public RenovationType Type
        {
            get; set;
        }

        public Renovation(int id, int firstRoomId, int secondRoomId, DateTime date, int duration, RenovationType type)
        {
            Id = id;
            FirstRoomId = firstRoomId;
            SecondRoomId = secondRoomId;
            Date = date;
            Duration = duration;
            Type = type;
        }

        public Renovation() { }


    }
}
